base/server/src/main/java/org/dogtagpki/server/PKIServerSocketListener.java

```diff
@@ -56,17 +60,39 @@
     public void alertReceived(SSLAlertEvent event) {
         try {
             SSLSocket socket = event.getSocket();
+            SSLFDProxy fdProxy = event.getFileDesc();
+	    JSSEngine engine = event.getEngine();
+
+            InetAddress clientAddress = null;
+            InetAddress serverAddress = null;
+            String clientIP = "Unknown";
+	    String serverIP = "Unknown"; 
```

Comment:
Are you having tab issues that's causing indentation differences?  You might want to fix those.